docs/api/api-reference.md

```diff
@@ -92,7 +92,7 @@ _Appears in:_
 | Field | Description |
 | --- | --- |
 | `secretRef` _string_ | SecretRef is the name of a Kubernetes secret in the consumer's (VDS/VSS/PKI/HCP) namespace which provides the HCP ServicePrincipal clientID, and clientKey. 
```

Comment:
```suggestion
| `secretRef` _string_ | SecretRef is the name of a Kubernetes secret in the consumer's (VDS/VSS/PKI/HCP) namespace which provides the HCP ServicePrincipal clientID, and clientSecret. 
```

Reply:
Good catch. Should be fixed with 33639dab5df1fc80e44f4a312af681a2a08391c1